X/XicTools/build_tarballs.jl

```diff
@@ -0,0 +1,130 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "XicTools"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/wrcad/xictools", "c7a50a5fcd71966730e45a5358b8507227ae098c"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/xictools/
+cp Makefile.sample Makefile
+sed -i "s|enable-itopok=no|enable-itopok=yes|g" Makefile
+sed -i "s|GFXLOC = --enable-gtk2=yes|#GFXLOC = --enable-gtk2=yes|g" Makefile
+#sed -i "s|^SUBDIRS =.*|SUBDIRS = xt_base adms KLU vl wrspice|g" Makefile
+sed -i "766c\#include <malloc.h>" xt_base/malloc/malloc-2.8.6.c
```

Comment:
This way of patching files with `sed` is very fragile, if the file is changed upstream then the line number doesn't match anymore and no one will tell you.  It's better to have proper patches and apply them with [`atomic_patch`](https://docs.binarybuilder.org/stable/build_tips/#utils_build_env), there are loads of similar examples in this repository.

Reply:
Totally agree, fixed this.